src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace/Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFrameworks)</TargetFramework>
```

Comment:
Nit: this is using a plural property to set a non-plural property, so could break in future. 

Also since this DLL is inserted into Roslyn, and really should track what they build, maybe it should remain directly specified?

Reply:
Switched to singular property, and I've added a comment about needing to be compatible with (details)